docs/adr/ADR-003-agent-first-control-plane.md

```
@@ -0,0 +1,19 @@
+# ADR-003: Agent-First Control Plane
+
+## Status
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)